### ISSUE TITLE
Raise Hand Support

### ISSUE THREAD

**Issue by dawalters1** [dawalters1/wolf.js#81]
10.14 introduced a stage feature called 'Raise Hands' in which users can request to join the stage, or someone of power can invite a user onto the stage

**Comment by dawalters1** [comment]
Will be available in 0.20.0

**Comment by dawalters1** [comment]
Introduces the following 

## Methods - Stage Helper
```JS
api.stage().getSlotRequestList(...) // Request a groups slot/mic request list
api.stage().requestSlot(...) // Bot will request a slot
api.stage().cancelSlotRequest(...) // Bot will cancel a slot request
api.stage().addSlotRequest(...) // Bot will request a user to join a specific slot
api.stage().removeSlotRequest(...) // Bot will cancel a addSlotRequest
api.stage().clearSlotRequestList(...) // Clear a groups slot/mic request lsit
```
## Events
```JS 
api.on('groupAudioRequestAdd', (group, request)=>{}) // A request was added
api.on('groupAudioRequestDelete', (group, request)=>{}) // A request was deleted
api.on('groupAudioRequestListClear', (group)=>{}) // The groups request list was cleared
api.on('groupAudioRequestExpire', (group, request)=>{}) // A request expired
```

#### Note: Method names will change before release as I am unhappy with the naming scheme
